awx/api/serializers.py

```diff
@@ -713,14 +714,17 @@ class UnifiedJobSerializer(BaseSerializer):
         read_only=True
     )
 
+    launched_by = serializers.SerializerMethodField(help_text=_('Object that launched the current job'))
+    ancestor_job = serializers.SerializerMethodField(help_text=_('The parent workflow that started the job, if not then ancestor_job returns self '))
+
     class Meta:
         model = UnifiedJob
         fields = ('*', 'unified_job_template', 'launch_type', 'status',
                   'execution_environment',
                   'failed', 'started', 'finished', 'canceled_on', 'elapsed', 'job_args',
                   'job_cwd', 'job_env', 'job_explanation',
                   'execution_node', 'controller_node',
-                  'result_traceback', 'event_processing_finished')
+                  'result_traceback', 'event_processing_finished', 'launched_by', 'ancestor_job')
```

Comment:
With the summary field added, we shouldn't need this anymore.

Reply:
I have removed it.